src/lib.rs

```diff
@@ -60,18 +60,23 @@ pub enum DisplayState {
     Editable,
 }
 
-/// Configure spacing and separators for the view
+///Controls the visual output of the `HexView` struct.
+///
+///There are various options which can be altered. For a detailed description of them, please see the fields below.
+///For the changes to apply, you need to use [`set_config`].
+///
+///[`set_config`]: struct.HexView.html#method.set_config
 #[derive(Debug, Clone, Copy)]
 pub struct HexViewConfig {
-    /// Number of bytes per line
+    /// Controls the number of bytes per line. It needs to be greater than 0 and equal or higher than the bytes_per group value. Default is 16
     pub bytes_per_line: usize,
-    /// Number of bytes per group
+    /// Controls the number of bytes per group. It needs to be greater than 0 equal or lower than bytes_per_line. Default is 1.
     pub bytes_per_group: usize,
-    /// Separation between groups
-    pub chars_per_spacing: usize,
-    /// Separator between the address and data
+    /// Controls the separator between the hex groups in the data output. Default is ` ` (1 char long)
+    pub byte_group_separator: &'static str,
+    /// Controls the separator between the address label and the hex output of the data. Default is ` : ` (3 chars long)
     pub addr_hex_separator: &'static str,
-    /// Separator between the hex and ascii representations
+    ///Controls the separator between the hex output and the ASCII representation of the data. Default is ` | ` (3 chars long)
     pub hex_ascii_separator: &'static str,
```

Comment:
I did the newlines in my [previous proposal](https://github.com/hellow554/cursive_hexview/pull/7#discussion_r551460615) on purpose. The first line is always a short description on what it does, then a newline and then a detailed description. See https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html (and search for "first").

Use the following as an example to modify the doc of **all** fields

```
Controls the number of bytes per line.

It needs to be greater than 0 and equal or higher than the bytes_per group value.
Default is 16.
```

You can run `cargo doc --open` localy to see what it would look like in the end. It helps me to get a better understanding on what is linked and what not.

Reply:
@palako please see this comment. Please note the empty line after the first line!